src/schoology-extractor/edfi_schoology_extractor/extract_facade.py

```diff
@@ -17,19 +17,33 @@
 from edfi_schoology_extractor import usage_analytics_facade
 import edfi_schoology_extractor.lms_filesystem as lms
 from edfi_schoology_extractor.helpers.sync import get_sync_db_engine
-from edfi_schoology_extractor.client_facade import SchoologyExtractFacade
+from edfi_schoology_extractor.client_facade import ClientFacade
+from edfi_lms_extractor_lib.helpers.decorators import catch_exceptions
 
 logger = logging.getLogger(__name__)
 
+# This variable facilitates temporary storage of output results from one GET
+# request that need to be used for creating another GET request.
+result_bucket: Dict[str, DataFrame] = {}
```

Comment:
This was previously in the middle of the file, makes more sense to be at the top.